pytorch_lightning/trainer/training_loop.py

```diff
@@ -396,6 +394,12 @@ def _process_training_step_output_1_0(self, training_step_output, split_batch):
             hiddens = training_step_output.pop('hiddens', None)
             result['extra'] = training_step_output
 
+            if loss is None:
+                model_ref = self.trainer.get_model()
+                func_name = "training_step_end" if is_overridden('training_step_end', model_ref) else "training_step"
+                m = f'The key `loss` should be present within {func_name} output. Existing keys: {[*training_step_output]}'
+                raise MisconfigurationException(m)
```

Comment:
```suggestion
                raise MisconfigurationException(
                    f'The key `loss` should be present within {func_name} output. Existing keys: {[*training_step_output]}'
                )
```
is there an advantage of having a message as separate var?